package.json

```diff
@@ -142,7 +148,7 @@
     "typescript": "^4.9.4",
     "webpack": "^5.88.2",
     "webpack-retry-chunk-load-plugin": "^3.1.1",
-    "wrangler": "^3.4.0",
+    "wrangler": "0.0.0-6ccc4fa6",
```

Comment:
Are you intentionally downgrading?

Reply:
yeah the latest wrangler version has some weird compatibilities with og so im downgrading back to the previous version (found the version hosted on npm so hopefully won't get taken down like the previous github link)